src/setup_runs/cmaq/read_config_cmaq.py

```diff
@@ -0,0 +1,98 @@
+import datetime
+
+from attrs import define, field
+import json
+from setup_runs.config_read_functions import boolean_converter, process_date_string
+
+
+def boolean_tuple(x):
+    return (boolean_converter(x))
```

Comment:
This doesn't return a tuple if that is what you intended?


The comma is important

```suggestion
    return (boolean_converter(x),)
```

Reply:
I also just saw that the output is not a tuple. But apparently it works either way. I'll take a closer look at it again.